src/server/viper.ts

```diff
@@ -38,149 +35,161 @@ try {
+                                .map((diagnostic) => {
+                                    const range: Range =
+                                        getMotokoSourceRange(
+                                            motokoPath,
+                                            diagnostic.range,
+                                        ) || defaultRange;
+                                    const message = resolveViperMessage(diagnostic);
+                                    return <Diagnostic>{
+                                        ...diagnostic,
+                                        message,
+                                        range,
+                                        source: motokoPath,
+                                        relatedInformation: [
+                                            {
+                                                // Viper source location
+                                                location: {
+                                                    uri,
+                                                    range: diagnostic.range,
+                                                },
+                                                message: 'view in context',
+                                            },
+                                        ],
+                                    };
+                                });
+                            const success = !!viperDiagnostics.length;
+                            const allDiagnostics: Diagnostic[] = [
+                                ...(mocViperCache.get(uri)?.diagnostics.filter(
+                                    // Only update type checking diagnostics for the original file
+                                    (d) => !d.source || d.source === motokoPath,
+                                ) || []),
+                                ...(success
+                                    ? viperDiagnostics
+                                    : [
+                                          {
+                                              message: '✅',
+                                              source: motokoPath,
+                                              severity: DiagnosticSeverity.Information,
+                                              range: defaultRange,
+                                          },
+                                      ]),
+                            ];
+                            sendDiagnostics(motokoPath, allDiagnostics);
+                        }
+                    } catch (err) {
+                        console.error(`Error while sending Viper diagnostics: ${err}`);
+                    }
+                },
+            );
 
-    connection.onNotification(
-        new rpc.NotificationType<{ uri: string }>('VerificationNotStarted'),
-        () => {
-            console.log('(Verification not started)');
-        },
-    );
+            // connection.onNotification(
```

Comment:
can we drop this?

Reply:
Sure. Also dropped the other unused logging callbacks.